IBC_WORKING_GROUP.md

```diff
@@ -0,0 +1,33 @@
+# IBC Working Group
+
+Join and contribute to the IBC working group 
+
+The evolution of IBC requires cross-industry collaboration to support developing standards that are secure, open, and deployable. The IBC working group is a growing team of contributors supporting the standardization of blockchain development from protocol level to applications. 
+
+Join the IBC [Working Group](https://zc.vg/sf/giAEr) email updates
+
+Join the IBC Telegram channel
+
+The founding IBC team
```

Comment:
If she is interested, I would invite @ancazamfir to join, her spec reviews have been really helpful.

Reply:
Yes I would love to if everyone is ok with it.